egs/yomdle_tamil/v1/local/yomdle/normalized_scoring/utils/normalize_common.py

```diff
@@ -0,0 +1,37 @@
+import sys
```

Comment:
If possible, please add a one-line description for all the scripts under `yomdle`.

Reply:
Thanks, I added the description by going through the scripts. Since the codes were small, I feel it is correct but I might have missed some details.